### PR TITLE
dojson: move force_force_list to inspirehep.utils

### LINKED ISSUE #1326: Move force_force_list to inspirehep/utils
The `force_list` util from DoJSON was being used outside of DoJSON rules to iterate over collections of one or more objects. This suggests that the `force_force_list` util should live in `inspirehep/utils` rather than `inspirehep/dojson/utils`.

This is an easy task, take it if you want something small to work while you wait for CI : )

### PR DESCRIPTION
**Pull request by tomaszgy**
* Closes #1326.

Signed-off-by: Tomasz Gargas <tomaszgy@gmail.com>

### CODE REVIEW

File: inspirehep/utils/bibtex.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 import re
 
-from inspirehep.dojson.utils import force_force_list
+from inspirehep.utils.helpers import force_force_list
```
Comment (jacquerie) [inline inspirehep/utils/bibtex.py]: Order: should go after `inspirehep.utils` and before `inspirehep.utils_record` (which should go before `record_getter`...)

File: inspirehep/modules/orcid/schema.py [hunk 2]
```diff
@@ -24,9 +24,10 @@
 
 import dojson
 
-from inspirehep.dojson.utils import force_force_list
 from inspirehep.utils.record import get_abstract, get_subtitle, get_title
 
+from inspirehep.utils.helpers import force_force_list
```
Comment (jacquerie) [inline inspirehep/modules/orcid/schema.py]: Order: should go before `inspirehep.utils.record`.

File: inspirehep/dojson/conferences/fields/bd1xx.py [hunk 3]
```diff
@@ -29,9 +29,11 @@
 from dojson import utils
 
 from ..model import conferences
-from ...utils import force_force_list
+
```
Comment (jacquerie) [inline inspirehep/dojson/conferences/fields/bd1xx.py]: I don't believe this space should be here.

**Comment by jacquerie** [comment]
LGTM, minus the three comments. Thanks!

**Comment by tomaszgy** [comment]
Corrected :smile:

**Comment by jacquerie** [comment]
Noticed one more thing: tests should be moved from `tests/unit/dojson/test_dojson_utils.py` to `tests/unit/utils/test_utils_helpers.py`.

**Comment by tomaszgy** [comment]
Sure, just updated.

**Comment by jacquerie** [comment]
Ouch, conflicts? I wasn't expecting them. @tomaszgy, are you busy? Otherwise I can take care of integrating this.

**Comment by tomaszgy** [comment]
I'll do it!

**Pull request closed without merging**